src/resources/Teams.php

```diff
@@ -17,7 +17,8 @@ public function __construct($api)
 			'deleteOne' => ['method' => 'DELETE', 'path' => '/teams/:teamId'],
 			'insertTask' =>  ['method' => 'PUT', 'path' => '/containers/teams/:teamId'],
 			'autoDispatch' =>  ['method' => 'POST', 'path' => '/teams/:teamId/dispatch'],
-			'getWorkerEta' => ['method' => 'GET', 'path' => '/teams/:teamId/estimate', 'queryParams' => true]
+			'getWorkerEta' => ['method' => 'GET', 'path' => '/teams/:teamId/estimate', 'queryParams' => true],
+			'listUnassignedTasks' => ['method' => 'GET', 'path' => '/teams/:teamId/tasks', 'queryParams' => true]
```

Comment:
We had named this method `getTasks` in the python wrapper so, we should keep the same names across the wrappers, shouldn't we?